doc/ref_model/chapters/glossary.md

```diff
@@ -98,3 +98,5 @@ To help guide the reader, this glossary provides an introduction to the terminol
 
 - **Virtualised Infrastructure Manager (VIM)**: responsible for controlling and managing the **NFVI** compute, storage and network resources.
 - **NFV Orchestrator (NFVO)**: manages the VNF lifecycle and **NFVI** resources (supported by the **VIM**) to ensure an optimised allocation of the necessary resources and connectivity.
+- **Platform**: A cloud capabilities type in which the cloud service user can deploy, manage and run customer-created or customer-acquired applications using one or more programming languages and one or more execution environments supported by the cloud service provider ([ITU](https://www.itu.int/rec/dologin_pub.asp?lang=e&id=T-REC-Y.3500-201408-I!!PDF-E&type=items)). 
```

Comment:
This is not Platform, but Platform capabilies type (PaaS) and I'm not sure why we need the PaaS definition here. 
Also there is a differentiation between PaaS and FaaS based on the interface between the platform and the applicaiton sand the ITU-T definition is more a FaaS, than a PaaS (according to my understanding).

Reply:
@CsatariGergely ITU differentiates between "Platform as a Service (PaaS)" and "platform capabilities type" -- Yes, the "platform" definition is for the "platform capabilities type."